packages/openapi-to-graphql/test/fixtures/example_oas4.json

```diff
@@ -533,6 +574,60 @@
             "type": "string"
           }
         }
+      },
+      "OneOfWithAllOfsAndLink": {
+        "oneOf": [
+          {
+            "$ref": "#/components/schemas/One"
+          },
+          {
+            "$ref": "#/components/schemas/Two"
+          }
+        ]
+      },
+      "Abstract": {
+        "type": "object",
+        "properties": {
+          "commonProp": {
+            "type": "string"
+          }
+        }
+      },
+      "OneProps": {
+        "type": "object",
+        "properties": {
```

Comment:
I was initially confused as to what the expected schema is from your OAS. I was expecting the member types to be object types but they were JSON instead. I thought there may have been an issue with the code but I later realized that your example was more for illustration. By giving the member types `properties`, they no longer resolve to JSON and your example become clearer.